### PR TITLE
Add new version in kanbanbot

### PR DESCRIPTION
**Pull request by tleon**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Please be specific when describing this change. What did you change? Why?
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Sponsor company   | PrestaShop SA
| How to test?      | CI green

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @tleon!

This is your first pull request on [kanbanbot](https://github.com/PrestaShop/kanbanbot/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

**Review by boherm — DISMISSED** [review]
it looks good, thanks @tleon!

**Comment by boherm** [comment]
Can you check CI please? 🙂

File: tests/Shared/Infrastructure/Provider/TranslationsCatalogProviderTest.php [hunk 1]
```diff
@@ -114,6 +114,32 @@ public static function provideTestGetCatalogVersionByPullRequest(): array
...
+                    id: new PullRequestId(
+                        repositoryOwner: 'PrestaShop',
+                        repositoryName: 'PrestaShop',
+                        pullRequestNumber: '1'
+                    ),
+                    labels: ['label'],
+                    approvals: [],
+                    targetBranch: '9.0.X'
+                ),
+                8,
```
Comment (jolelievre) [inline tests/Shared/Infrastructure/Provider/TranslationsCatalogProviderTest.php:128]:
```suggestion
                9,
```
Seems like the expected value should be 9 no?

**Review by boherm — APPROVED** [review]
Thanks @tleon! LGTM.

**Pull request merged**